src/operators/rastervectorjoin.md

```diff
@@ -0,0 +1,77 @@
+# RasterVectorJoin
+
+The `RasterVectorJoin` operator allows combining a single vector input and multiple raster inputs.
+For each raster input, a new column is added to the collection from the vector input.
+The new column contains the value of the raster at the location of the vector feature.
+For features covering multiple pixels like `MultiPoints` or `MultiPolygons`, the value is calculated using an aggregation function selected by the user.
+The same is true if the temporal extend of a vector feature covers multiple raster time-steps.
+More details are described below.
+
+**Example**:
+You have a collection of aggricaltural fields (`Polygons`) and a collection of raster images containing the monthly NDVI value for each pixel.
+The `RasterVectorJoin` operator allows you to do this.
+You now have multiple options to calculate the NDVI value for each field.
+For a quick evaluation, you can use the `first` aggregation function.
```

Comment:
Maybe first state what it is good for in general.
Then say, that it is cheap and also good for exploratory use.